OsmAnd/res/values/strings.xml

```diff
@@ -11,21 +11,21 @@
 	Thx - Hardy
 -->
 
-    <string name="switch_mode_by_tapping_on_widget">Switch the mode by tapping on the widget.</string>
-    <string name="map_widget_sunset_desc">Shows the time of the next sunset or the time left until sunset.</string>
-    <string name="map_widget_sunrise_desc">Shows the time of the next sunrise or the time left until sunrise.</string>
+    <string name="switch_mode_by_tapping_on_widget">Switch mode by tapping the widget.</string>
```

Comment:
Use article here, as we do elsewhere

Reply:
"On the widget" is use everywhere else.
The question is whether to use "… by tapping on" or "Tapping on the widget".
I would use "Tap the widget …", but that wasn't accepted before.